eqcorrscan/tests/match_filter_test.py

```diff
@@ -408,49 +408,20 @@ def test_catalog_extraction(self):
             match_filter(template_names=self.template_names,
                          template_list=self.templates, st=self.st,
                          threshold=8.0, threshold_type='MAD',
-                         trig_int=6.0, plotvar=False, plotdir='.',
+                         trig_int=6.0, plot=False, plotdir='.',
                          cores=1, extract_detections=True, output_cat=True)
         self.assertEqual(len(detections), 4)
         self.assertEqual(len(detection_streams), len(detections))
         self.assertEqual(len(detection_streams), len(det_cat))
 
-    def test_same_detections_individual_and_parallel(self):
```

Comment:
Was there a reason for removing this test?

Reply:
Thanks a lot for catching this. I got a bit confused because I had added two tests related to PR #439, of which one you already had merged into develop while the other one was part of the open PR. So I ended up thinking that my git history was messed up as it contained both tests for some reason; while it should have only contained the one which had already been merged. I reverted the last commit that removed `test_same_detections_individual_and_parallel` now, so the PR here should show only changes compared to the `develop`-branch now.